src/Estimators/include/BipedalLocomotion/Estimators/FloatingBaseEstimatorIO.h

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file FloatingBaseEstimatorIO.h
+ * @authors Prashanth Ramadoss
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_ESTIMATORS_FBE_IO_H
+#define BIPEDAL_LOCOMOTION_ESTIMATORS_FBE_IO_H
+
+#include <Eigen/Dense>
+
+namespace BipedalLocomotion
+{
+namespace Estimators
+{
+namespace FloatingBaseEstimators
+{
+/**
+* @brief Struct holding the elements of the state representation
+*
+*/
+struct InternalState
+{
+
+
```

Comment:
Probably you don't need this space 😄

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/79/commits/f46bd5b271b6e7ea77da83c41b90178157aac069